matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/SessionModule.kt

```diff
@@ -218,10 +219,12 @@ internal abstract class SessionModule {
         fun providesOkHttpClient(@UnauthenticatedWithCertificate okHttpClient: OkHttpClient,
                                  @Authenticated accessTokenProvider: AccessTokenProvider,
                                  @SessionId sessionId: String,
+                                 lowBandwidthInterceptor: LowBandwidthInterceptor,
                                  @MockHttpInterceptor testInterceptor: TestInterceptor?): OkHttpClient {
             return okHttpClient
                     .newBuilder()
                     .addAccessTokenInterceptor(accessTokenProvider)
+                    .addInterceptor(lowBandwidthInterceptor)
```

Comment:
Why do you add this interceptor to the Authenticated okHttpClient and not instead to the unauthenticated httpClient?

Reply:
because confusingly the authenticated client is not derived from the unauthenticated client. So if you just add it to the unauthenticated client you lose the interceptor after logging in.

Reply:
This is strange. Authenticated client should be derived from Unauthenticated client. Let me double check